contrib/hooks/post-receive/lib/git_multimail.py

```diff
@@ -1,5 +1,8 @@
-#! /usr/bin/env python2
```

Comment:
I don't think that should be part of the plugin. You can simply add it outside the repository.